src/Plugin/EntityReferenceSelection/OgSelection.php

```diff
@@ -37,11 +37,15 @@ class OgSelection extends DefaultSelection {
    * @return DefaultSelection
    */
   public function getSelectionHandler() {
-    $field = $this->getConfiguration('field');
-
-    if ($field) {
-      return \Drupal::service('plugin.manager.entity_reference_selection')->getSelectionHandler($field);
```

Comment:
> So is it an issue with Og::createField?

yes, we never handled those parts (it was in old code, but was removed in the Og::createField() cleanup)